modules/site-health/admin-site-health.php

```diff
@@ -0,0 +1,272 @@
+	 * @since   2.8
+	 */
+	protected function exclude_options_key() {
+		return array(
+			'uninstall',
+			'first_activation',
+		);
+	}
+
+	/**
+	 * Return a list of key to exclude from site health informations.
+	 *
+	 * @return array list of language key to ignore
+	 * @since   2.8
+	 */
+	protected function exclude_lang_key() {
+		return array(
+			'flag',
+			'host',
+			'taxonomy',
+			'description',
+			'parent',
+			'filter',
+			'custom_flag',
+		);
+	}
+
+	/**
+	 * Add Polylang Options to Site Health Informations tab.
+	 *
+	 * @param array $debug_info array options to display.
+	 * @return array
+	 * @since   2.8
+	 */
+	public function info_options( $debug_info ) {
+		$options = get_option( 'polylang' );
+		$fields = array();
+		foreach ( $options as $key => $value ) {
+			if ( in_array(
+				$key,
+				$this->exclude_options_key()
+			)
+				) {
+				continue;
+			}
+			if ( ! is_array( $value ) ) {
+				if ( false === $value ) {
+					$value = '0';
+				}
+				$fields[ $key ]['label']   = $key;
+				$fields[ $key ]['value']   = $value;
+			} else {
+				if ( empty( $value ) ) {
+					$fields[ $key ]['label']   = $key;
+					$fields[ $key ]['value']   = '0';
+				} else {
+					switch ( $key ) {
+						case 'post_types':
+							$fields[ $key ]['label'] = $key;
+							$fields[ $key ]['value'] = implode( ', ', $this->model->get_translated_post_types() );
+							break;
+						case 'taxonomies':
+							$fields[ $key ]['label'] = $key;
+							$fields[ $key ]['value'] = implode(
+								', ',
+								$this->model->get_translated_taxonomies()
+							);
+							break;
+						case 'nav_menus':
+							$current_theme = get_stylesheet();
+							foreach ( $value[ $current_theme ] as $location => $lang ) {
+								// translators: placeholder is the menu location name
+								$fields[ $location ]['label'] = sprintf( __( 'Menu: %s', 'polylang' ), $location );
+								array_walk(
+									$lang,
+									function ( &$value, $key ) {
+										$value = "$key:$value";
+									}
+								);
+								$fields[ $location ]['value'] .= implode( ' | ', $lang );
+							}
+							break;
+						case 'media':
+							array_walk(
+								$value,
+								function ( &$value, $key ) {
+									$value = "$key: $value";
+								}
+							);
+							$fields[ $key ]['label'] = '';
+							$fields[ $key ]['value'] .= implode( ',', $value );
+							break;
+						default:
+							$fields[ $key ]['label']   = $key;
+							$fields[ $key ]['value']   = implode( ', ', $value );
+							break;
+					}
+				}
+			}
+		}
+
+		$debug_info['polylang'] = array(
+			'label'    => __( 'Polylang Options', 'polylang' ),
+			'fields' => $fields,
+		);
+
+		return $debug_info;
+	}
+
+	/**
+	 * Add Polylang Languages settings to Site Health Informations tab.
+	 *
+	 * @param array $debug_info array options to display.
+	 * @return array
+	 * @since   2.8
+	 */
+	public function info_languages( $debug_info ) {
+		$languages = PLL()->model->get_languages_list();
```

Comment:
You already have access to this function from the `$model` property: https://github.com/polylang/polylang/pull/522/files#diff-cb22506afd9748f751e395fe4f93c2e7R14